# Copyright (C) 2021 LEIDOS.
#
# Licensed under the Apache License, Version 2.0 (the "License"); you may not
# use this file except in compliance with the License. You may obtain a copy of
# the License at
#
# http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
# License for the specific language governing permissions and limitations under
# the License.

# Double: The time allocated for system startup. Validation begins after this much time has passed.
# Units: s
startup_duration: 60.0

# Double: The time allocated for validation; once startup is complete, validation is complete after this much time has passed.
# Units: s
validation_duration: 60.0

# List of String: Guidance Strategic Plugins that will be validated by this node
guidance_strategic_plugins:
  - RouteFollowing 

# List of String: Guidance Tactical Plugins that will be validated by this node
guidance_tactical_plugins:
  - InLaneCruisingPlugin 
  - StopandWaitPlugin 
  - CooperativeLaneChangePlugin 
  - UnobstructedLaneChangePlugin 
  - YieldPlugin 

# List of String: Guidance Control-Wrapper Plugins that will be validated by this node
guidance_control_plugins:
  - Pure Pursuit 